junit/junit.bzl

```diff
@@ -2,31 +2,23 @@ load(
     "//scala:scala_cross_version.bzl",
     _default_maven_server_urls = "default_maven_server_urls",
 )
-load(
-    "@io_bazel_rules_scala//scala:scala_maven_import_external.bzl",
-    _scala_maven_import_external = "scala_maven_import_external",
-)
+load("//third_party/repositories:repositories.bzl", "repositories")
 
-def junit_repositories(maven_servers = _default_maven_server_urls()):
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_junit_junit",
-        artifact = "junit:junit:4.12",
-        artifact_sha256 = "59721f0805e223d84b90677887d9ff567dc534d7c502ca903c0c2b17f05c116a",
-        licenses = ["notice"],
-        server_urls = maven_servers,
+def junit_repositories(
+        maven_servers = _default_maven_server_urls(),
```

Comment:
`maven_servers` is not passed to `repositories`

Reply:
Thanks for catching, @simuons!